src/components/v2v3/V2V3Project/V2V3PayButton/V2V3ConfirmPayModal/V2V3PayForm.tsx

```diff
@@ -146,7 +148,12 @@ export const V2V3PayForm = ({
               <Space>
                 <Switch
                   checked={customBeneficiaryEnabled}
-                  onChange={setCustomBeneficiaryEnabled}
+                  onChange={() => {
```

Comment:
**suggstion** think we should still rely on the state value that the component is giving us. WDYT?

```suggestion
                  onChange={(enabled) => {
```

Reply:
Yeah good call, sounds good